learn/safe-core/safe-core-account-abstraction-sdk/auth-kit.md

```diff
@@ -4,51 +4,123 @@ The [Auth kit](https://github.com/safe-global/safe-core-sdk/tree/main/packages/a
 
 Note: The Auth kit creates a [Signing Account, not a Smart Account](/learn/what-is-a-smart-contract-account.md#smart-accounts-vs-signing-accounts).
 
-The quick start guide below shows you how to sign transactions using your Signing Account. You can sign transactions using your Signing Account to create a Smart Account.
+The quick start guide below shows you how to create a Signing Account using your email or social media account. Once authenticated, you can sign transactions and interact with any Safe Smart Accounts you own.
 
 ## Quickstart
 
-### Prerequisites
+## Prerequisites
 
 - [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
-- [Create a Web3Auth account](https://web3auth.io) to use Web3Auth as the authentication provider.
+- [Create a Web3Auth account](https://web3auth.io)
+
+## Example
+
+See an [Auth Kit example in the Safe Core SDK](https://github.com/safe-global/safe-core-sdk/blob/7959821ab08c96cf3babb9ed906c01d644ac49f4/packages/auth-kit/example/src/App.tsx#L33) and [Pull Request 12](https://github.com/5afe/safe-space/pull/12) in [Safe Space](https://github.com/5afe/safe-space).
+
+
+{% embed url="https://user-images.githubusercontent.com/9806858/234088191-441c1c18-5922-47a4-a4dd-44fb383a54d6.mp4" %}
+[Watch video demo](https://user-images.githubusercontent.com/9806858/234088191-441c1c18-5922-47a4-a4dd-44fb383a54d6.mp4)
+{% endembed %}
+
+
 
 ### Install dependencies
 
 ```bash
 yarn add @safe-global/auth-kit @web3auth/base @web3auth/modal @web3auth/openlogin-adapter
 ```
 
-### How to use
+If you run `yarn start` and see an error similar to `Module not found: Error: Can't resolve 'crypto'`, see Web3Auth's [Webpack 5 Polyfills Issue](https://web3auth.io/docs/troubleshooting/webpack-issues) for more context on how to fix or [commit 85ccd22](https://github.com/5afe/safe-space/pull/12/commits/85ccd22b8528d7eacd013a8e3f5bb0093c85b081) for the necessary code changes to fix the Polyfill issues.
+
+Note: You might also need to also install `browserify-zlib` (`yarn add browserify-zlib`), which is not included in the Web3Auth documentation.
```

Comment:
Out of curiosity, I had never installed this to solve this issue. Is this specific to create-react-app and Web3Auth? I’m using vite and a plugin to address this known issue.

Reply:
Yes, this is a troubleshooting step. Some users may face this issue some may not face this issue. I faced this issue. Which is why I say "**if** you run `yarn start` and see an error"